features/step_definitions/visitor_can_register_for_an_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Visitor can register for an account
+
+    As a Visitor,
+    In order to access unique features on the news app,
+    I would like to be able to register for an account on the site.
+
+    Background:
+        Given the following user exists
+        | name | email          | password | password_confirmation |
+        | Bill | bill@email.com | password | password              |
```

Comment:
Is there any need for this background step

Reply:
Now there is

Reply:
We have added tests for the sad path so that's why it is needed now